nav2_costmap_2d/plugins/costmap_filters/costmap_filter.cpp

```diff
@@ -78,7 +78,7 @@ void CostmapFilter::onInitialize()
     // Get parameters
     node->get_parameter(name_ + "." + "enabled", enabled_);
     filter_info_topic_ = node->get_parameter(name_ + "." + "filter_info_topic").as_string();
-    double transform_tolerance;
+    double transform_tolerance {};
```

Comment:
For all these: set to zero

Reply:
20acdca5436f2dcb918f9cceeb35730410b3b26b